src/widgets/communication/ui/communication.tsx

```diff
@@ -9,81 +10,63 @@ import {
 
-interface RequiredProps {
+const cx = classNames.bind(styles);
+
+type CommunicationProps = {
   courseName: CourseNamesChannels;
   lang?: 'ru' | 'en';
-}
-
-const localizedContent = {
-  en: {
-    title: 'Communication',
-    subTitle: 'Discord is the main communication channel in RS School',
-    firstParagraphFirstHalf: 'Here is link for the',
-    discordLink: 'course Discord server',
-    firstParagraphSecondHalf: ', where you can see latest news and chat with students.',
-    secondParagraphFirstHalf: 'There are channels in',
-    telegramLink: 'Telegram',
-    secondParagraphSecondHalf:
-      ' for discussing events related to your location. For example, offline lectures or just informal chats among students from the same location.',
-    thirdParagraphFirstHalf: 'Please read the information about communication in RS School in the',
-    rsDocsLink: 'RS Docs',
-    thirdParagraphSecondHalf: ', where you can find rules, descriptions of channels, FAQ.',
-  },
-  ru: {
-    title: 'Общение',
-    subTitle: 'Дискорд — основной способ общения в RS School',
-    firstParagraphFirstHalf: 'Вот ссылка на',
-    discordLink: 'Дискорд сервер курса',
-    firstParagraphSecondHalf:
-      ', где вы можете посмотреть последние новости, задать вопросы и общаться со студентами.',
-    secondParagraphFirstHalf: 'Также есть каналы в',
-    telegramLink: 'Телеграм',
-    secondParagraphSecondHalf:
-      ' для обсуждения мероприятий, относящихся к вашему городу. Например, офлайн лекции или просто для общения студентов из одной локации.',
-    thirdParagraphFirstHalf: 'Обязательно прочитайте информацию об общении в RS School в',
-    rsDocsLink: 'RS Docs',
-    thirdParagraphSecondHalf: ', где вы можете найти правила, описание каналов, FAQ.',
-  },
 };
 
-export const Communication = ({ courseName, lang = 'en' }: RequiredProps) => {
+export const Communication = ({ courseName, lang = 'en' }: CommunicationProps) => {
+  const {
+    title,
+    subTitle,
+    firstParagraphFirstHalf,
+    discordLink,
+    firstParagraphSecondHalf,
+    secondParagraphFirstHalf,
+    telegramLink,
+    secondParagraphSecondHalf,
+    thirdParagraphFirstHalf,
+    rsDocsLink,
+    thirdParagraphSecondHalf,
+  } = communicationText[lang];
+
   return (
-    <section className="communication container">
-      <article className="communication content info-wrapper">
-        <WidgetTitle mods="asterisk">{localizedContent[lang].title}</WidgetTitle>
-        <div className="column-2">
-          <figure className="disclogo-wrapper">
-            <DiscordLogo />
+    <section className={cx('container')}>
+      <article className={cx('content', 'communication-content')}>
+        <WidgetTitle mods="asterisk">{title}</WidgetTitle>
+        <div className={cx('communication-text')}>
+          <figure className={cx('discord-logo-wrapper')}>
+            <Image src={discordLogo} alt="discord logo" />
           </figure>
           <div>
```

Comment:
```suggestion
          <article className={cx('communication-description')}>
```

Reply:
If its needed, I can change it in the article, but why do we need communication-description, Where is it in the code?

Reply:
In code it is below, on UI here it is 
![image](https://github.com/user-attachments/assets/5e8bd97b-e655-4204-9ea3-bff3f974ded0)